README.md

```diff
@@ -125,3 +125,20 @@ Configuring Shibboleth
 * visit '/Shibboleth.sso/Metadata' to download and review the metadata, rename to a filename of your preference
 * upload the SP metadata to http://testshib.org/register.html or your Identiy Provider (IdP)
 * login by clicking on 'Sign into Shibboleth' and choosing one of the available identities
+
+Audit Fix
+---
+* To fix data with triple files:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'triples','sample-sparql.txt'```
+* To remove irrelevant theses dates:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'thesis'```
```

Comment:
Suggestions on how to test these? Obviously my problem is that I don't have the right data loaded.
```
bin/fix/fix-audit.rb:73:in `find_objects': undefined method `length' for nil:NilClass (NoMethodError)
	from bin/fix/fix-audit.rb:111:in `fix_theses_related_dates'
	from bin/fix/fix-audit.rb:180:in `<main>'
```